include/Gaffer/Private/IECorePreview/LRUCache.inl

```diff
@@ -624,8 +624,15 @@ void LRUCache<Key, Value, Policy, GetterKey>::clear()
 template<typename Key, typename Value, template <typename> class Policy, typename GetterKey>
 void LRUCache<Key, Value, Policy, GetterKey>::setMaxCost( Cost maxCost )
 {
-	m_maxCost = maxCost;
-	limitCost( maxCost );
+	if( maxCost > m_maxCost )
```

Comment:
Could you make the equivalent PR for Cortex too please? I'd like to keep them in sync...